config/versions.yaml

```diff
@@ -16,6 +16,15 @@ installers:
       - 'debian9'
       - 'ubuntu1604'
 
+  - foreman: '1.17'
+    katello: '3.6'
+    puppet: 5
+    boxes:
+      - 'centos7'
+      - 'debian8'
```

Comment:
I *think* foreman 1.17 does not support Debian 8 anymore, but @mmoll can confirm that

Reply:
It was indeed dropped.